### PR TITLE
Update test frequency of ocp 4.13 z-stream upgrade

### PR DESCRIPTION
**Pull request by JianLi-RH**
/cc @jianlinliu @jiajliu @shellyyang1989 

```
[jianl@jianl-thinkpadt14gen4 release]$  grep -e "-f28" ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13.yaml
- as: gcp-ipi-oidc-rt-fips-f28
- as: ibmcloud-ipi-private-fips-f28
- as: nutanix-ipi-proxy-fips-f28
- as: vsphere-upi-encrypt-f28
[jianl@jianl-thinkpadt14gen4 release]$ 
[jianl@jianl-thinkpadt14gen4 release]$ grep -e "-f28" ci-operator/config/openshift/openshift-tests-private/openshift-openshift-te
sts-private-release-4.13__multi-nightly-4.13-upgrade-from-stable-4.13.yaml
- as: aws-ipi-proxy-sts-arm-f28
- as: azure-ipi-ovn-ipsec-azurefile-csi-arm-f28
[jianl@jianl-thinkpadt14gen4 release]$ 
```

### CODE REVIEW

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13.yaml [hunk 1]
```diff
@@ -257,7 +257,7 @@ tests:
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-vsphere-ipi-proxy-workers-rhel8
 - as: vsphere-upi-encrypt-f28
-  cron: 22 16 26 * *
+  cron: 4 20 28 * *
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13.yaml:260]: One more `f28` there

Comment (JianLi-RH) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13.yaml:260]: It's expected, ideally we want cover all platform and each platform has one f28 job:
```
[jianl@jianl-thinkpadt14gen4 release]$ grep -e "-f28" ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__*-4.13-upgrade-from-stable-4.13.yaml
ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13.yaml:- as: gcp-ipi-oidc-rt-fips-f28
ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13.yaml:- as: ibmcloud-ipi-private-fips-f28
ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13.yaml:- as: nutanix-ipi-proxy-fips-f28
ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13.yaml:- as: vsphere-upi-encrypt-f28
ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__multi-nightly-4.13-upgrade-from-stable-4.13.yaml:- as: aws-ipi-proxy-sts-arm-f28
ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__multi-nightly-4.13-upgrade-from-stable-4.13.yaml:- as: azure-ipi-ovn-ipsec-azurefile-csi-arm-f28
[jianl@jianl-thinkpadt14gen4 release]$ 
```

Comment (liangxia) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13.yaml:260]: Ack.
Maybe some explanation next time ? We are not aware of your subteam's decision.

**Pull request merged**